gradle-failure-reports/src/main/java/com/palantir/gradle/failurereports/FailureReportsExtension.java

```diff
@@ -34,5 +38,6 @@ public FailureReportsExtension() {
                 .convention(getProjectLayout().getBuildDirectory().file("failure-reports/build-TEST.xml"));
         getFailureReportCompileOutputFile()
                 .convention(getProjectLayout().getBuildDirectory().file("failure-reports/build-compile-TEST.xml"));
+        getIgnoredTasks().add(Test.class);
```

Comment:
I was just coming here to say this. I think the place where you consume from the extension, should have `Test` as a static location, where you then combine it with the user provided classes. We do this in dev-env and also gradle-sls-docker so users can't just wreck themselves.